lessons/linuxadmin/installation/apple-silicon.md

```diff
@@ -0,0 +1,132 @@
+
+Jestli už víš, že Tvé zařízení používá procesor Intel, můžeš přestat číst tento návod a přejdi [sem]({{ subpage_url('virtualbox.md') }}).
+
+Pokud jsi stále na pochybách, raději zkontroluj jaké zařízení vlastně máš.
+
+### Jak poznat který procesor máš
+Cvakni na logo jablka.
+{{ figure(
+    img=static('apple-silicon/apple-logo.png'),
+    alt='Ikonka jablka v horní liště',
+) }}
+Vyber možnost "O tomto Macu"
+{{ figure(
+    img=static('apple-silicon/about-mac.png'),
+    alt='Výběr "O tomto Macu" z lišty',
+) }}
+Vyskočí na Tebe okno s informacemi o Tvém zařízení.
+Tebe zajímá kolonka "Čip". 
+{{ figure(
+    img=static('apple-silicon/cpu-info.png'),
+    alt='Okno s informacemi o procesoru',
+) }}
+- Pokud obsahuje nějakou zmínku o "Intelu", pak se tě tento návod netýká a můžeš pokračovat [na tomto odkaze]({{ subpage_url('virtualbox.md') }}).
+- Pokud zmiňuje Apple a řadu M jako na obrázku, tak čti dál...
+
+## Instalace UTM
+Vzhledem k omezením spojených s architekturou Apple Silicon procesorů bude nejlepší použít nástroj zvaný [UTM]({{ subpage_url('https://mac.getutm.app') }}).
+
+UTM je svobodný software, což mimo jiné znamená, že je zcela zdarma.
+
+Pokud náhodou používáš balíčkovací nástroj brew, UTM nainstaluješ v terminálu pomocí příkazu 
+`brew install --cask utm` a můžeš pokračovat [další kapitolou]({{ subpage_url('#vytvoření-virtuálního-počítače') }}).
+
+Jinak UTM můžeš stáhnout z [oficiálních stránek]({{ subpage_url('https://mac.getutm.app') }}) kliknutím na tlačítko "Download".
+{{ figure(
+    img=static('apple-silicon/download-utm.png'),
+    alt='Stažení UTM',
+) }}
+
+Stažení je nutné povolit cvaknutím na "Povolit".
+{{ figure(
+    img=static('apple-silicon/utm-download-window.png'),
+    alt='Povolení stažení UTM',
+) }}
+
+Mělo by začít stahování a po chvíli by se v doku ve spodní části obrazovky měl objevit soubor nazvaný "UTM", případně "UTM.dmg".
+{{ figure(
+    img=static('apple-silicon/utm-in-dock.png'),
+    alt='Stažený utm.dmg soubor v doku',
+) }}
+
+Na ten cvakni a vyskočí na tebe instalační okno.
+{{ figure(
+    img=static('apple-silicon/utm-install.png'),
+    alt='Instalace UTM',
+) }}
+Nyní přetáhni UTM do složky "Applications".
+Tím by měla být instalace dokončena.
+
+## Vytvoření virtuálního počítače
+{{ figure(
+    img=static('apple-silicon/utm-create-vm.png'),
+    alt='Vytvoření virtuálního stroje v UTM',
+) }}
+{{ figure(
+    img=static('apple-silicon/utm-virtualize.png'),
+    alt='Výběr virtualizace v UTM',
+) }}
+{{ figure(
+    img=static('apple-silicon/utm-choose-linux.png'),
+    alt='Výběr Linuxu v UTM',
+) }}
+{{ figure(
+    img=static('apple-silicon/utm-setup-vm.png'),
+    alt='Nastavení virtuálního stroje v UTM',
+) }}
+Obraz by už měl být stažen z [dřívější kapitoly]({{ subpage_url('index.md#stažení-obrazu-') }}). 
+Pokud si nic takového nepamatuješ, znovu si přečti [dřívější kapitolu]({{ subpage_url('index.md#stažení-obrazu-') }}) a poté se znovu vrať sem.
```

Comment:
```suggestion
Obraz by už měl být stažen z [dřívější kapitoly]({{ subpage_url('index.md#stažení-obrazu-') }}). 
Pokud si nic takového nepamatuješ, znovu si přečti [dřívější kapitolu]({{ subpage_url('index.md#stažení-obrazu-') }}) a poté se znovu vrať sem.
```
Nemá tam být ta přípona `.md` a ty kotvy se nedělají automaticky. Jinde v kódu je to jako:
```
{{ anchor('install-system') }}
# Instalace systému
```
a pak se to referencuje
```
Pokračuj [instalací systému]({{ subpage_url('index#install-system') }}).
```